ghcup-prereleases-0.0.8.yaml

```diff
@@ -1864,12 +1864,125 @@ ghcupDownloads:
+              dlHash: f25c93a1fae2dde327dc34c72002414373871caa4253b7b90bb58484a8ec92fc
+              dlOutput: ghc-9.10.0.20240426-x86_64-linux-fedora33.tar.xz
+              dlSubdir: ghc-9.10.0.20240426-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240426/ghc-9.10.0.20240426-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc9101rc1-x86_64-centos7
+          Linux_Mint:
+            < 20: &ghc9101rc1-x86_64-ubuntu18_04
+              dlHash: 0154dbbbe77b7524c01c4defd6d885e948221943287009baa3cd0c151623f233
+              dlOutput: ghc-9.10.0.20240426-x86_64-linux-ubuntu18_04.tar.xz
+              dlSubdir: ghc-9.10.0.20240426-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240426/ghc-9.10.0.20240426-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc9101rc1-x86_64-ubuntu20_04
+              dlHash: 1e46d968507f3991d8103abfbdc19f0dbfccc60676ee2a686ea60523524741aa
+              dlOutput: ghc-9.10.0.20240426-x86_64-linux-ubuntu20_04.tar.xz
+              dlSubdir: ghc-9.10.0.20240426-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240426/ghc-9.10.0.20240426-x86_64-ubuntu20_04-linux.tar.xz
+          Linux_RedHat:
+            unknown_versioning: *ghc9101rc1-x86_64-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 18 ): *ghc9101rc1-x86_64-deb9
+            ( >= 18 && < 19 ): *ghc9101rc1-x86_64-ubuntu18_04
+            unknown_versioning: *ghc9101rc1-x86_64-ubuntu20_04
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: 775065ec1f59fc7ddf99418b7fe7cbf80289d358e1e8bb30ba6575542716837a
+              dlOutput: ghc-9.10.0.20240426-x86_64-linux-rocky8.tar.xz
+              dlSubdir: ghc-9.10.0.20240426-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240426/ghc-9.10.0.20240426-x86_64-rocky8-linux.tar.xz
+          Windows:
+            unknown_versioning:
+              dlHash: 499210e6829b7ab062233ba550928bfa33de475258b00a52dbca4945ac2ff05d
+              dlOutput: ghc-9.10.0.20240426-x86_64-windows.tar.xz
+              dlSubdir: ghc-9.10.0.20240426-x86_64-unknown-mingw32
+              dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240426/ghc-9.10.0.20240426-x86_64-unknown-mingw32.tar.xz
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlHash: 87b1388e941336b00d165550b08a97edb7991d669fef28b594924df7d591a425
+              dlOutput: ghc-9.10.0.20240426-aarch64-darwin.tar.xz
+              dlSubdir: ghc-9.10.0.20240426-aarch64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.10.0.20240426/ghc-9.10.0.20240426-aarch64-apple-darwin.tar.xz
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: 7f8d1f4e54164a9b7ac08b907745ac5ba74b979fa2e063cf26365187098c3098
+              dlOutput: ghc-9.10.0.20240426-aarch64-linux-deb10.tar.xz
```

Comment:
Can we avoid `dlOutput` when it's essentially the same as the one in `dlUri`? It's just noise otherwise.

Reply:
Sure.